tools/sphere.c

```diff
@@ -0,0 +1,108 @@
+// Mystical Sphere of Slooping
+// Gon
+// 2017-04
+
+inherit NewThingCode;
+inherit AnsiTellCode;
+inherit "/usr/gon/tools/sloop";
```

Comment:
If you refer to my comment about creating a #define for the sloop directory in sloop.c, we could put it in a separate .h file that both sloop.c and sphere.c #include. Then sphere.c could use it here.